huxley/www/views.py

```diff
@@ -21,9 +21,17 @@ def index(request):
     if request.user.is_authenticated():
         user_dict = UserSerializer(request.user).data
 
+    session = Conference.objects.all().aggregate(Max('session'))['session__max']
+    conference = Conference.objects.get(session=session)
```

Comment:
I feel like this implementation is bug prone. If we don't explicitly document that we are using the maximum conference then users could see some unexpected behavior. I am okay leaving it as is but we would need to make it clear. One thing I have been thinking about is to create a properties file that stores all these properties about the app then injects them into the backend. That will take some more thought but this would be a place where that is useful.